src/app/javascript/editor.js

```diff
@@ -0,0 +1,8 @@
+require("codemirror/mode/ruby/ruby");
+var CodeMirror = require("codemirror")
+window.addEventListener('load', function() {
+    var editor = CodeMirror.fromTextArea(document.getElementById('code'), {
```

Comment:
Question: will this get executed in every page? because if so I would expect it to raise an error in all other pages that doesn't include this `#code` tag